config/templates/operator.yaml.in

```diff
@@ -0,0 +1,302 @@
+      - ""
+    resources:
+      - services/finalizers
+    verbs:
+      - "*"
+  - apiGroups:
+      - ""
+    resources:
+      - services/status
+    verbs:
+      - "*"
+  - apiGroups:
+      - crdb.cockroachlabs.com
+    resources:
+      - crdbclusters
+    verbs:
+      - "*"
+  - apiGroups:
+      - crdb.cockroachlabs.com
+    resources:
+      - crdbclusters/status
+    verbs:
+      - "*"
+  - apiGroups:
+      - policy
+    resources:
+      - poddisruptionbudgets
+    verbs:
+      - "*"
+  - apiGroups:
+      - policy
+    resources:
+      - poddisruptionbudgets/finalizers
+    verbs:
+      - "*"
+  - apiGroups:
+      - policy
+    resources:
+      - poddisruptionbudgets/status
+    verbs:
+      - "*"
+  - verbs:
+      - use
+    apiGroups:
+      - security.openshift.io
+    resources:
+      - securitycontextconstraints
+    resourceNames:
+      - nonroot
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cockroach-operator-sa
+  namespace: default
+  annotations:
+  labels:
+    app: cockroach-operator
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: cockroach-operator-default
+  labels:
+    app: cockroach-operator
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: cockroach-operator-role
+subjects:
+  - name: cockroach-operator-sa
+    namespace: default
+    kind: ServiceAccount
+
+# Operator Deployment Definition:
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cockroach-operator
+  namespace: default
+  labels:
+    app: cockroach-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: cockroach-operator
+  template:
+    metadata:
+      labels:
+        app: cockroach-operator
+    spec:
+      serviceAccountName: cockroach-operator-sa
+      containers:
+        - name: cockroach-operator
+          image: cockroachdb/cockroach-operator:{{ .OperatorVersion }}
+          imagePullPolicy: IfNotPresent
+          # uncomment this to enable different new features
+          # args: ["-feature-gates","PartitionedUpdate=true,UseDecommission=true,Upgrade=false, CrdbVersionValidator=false"]
+          # this is for openshift only
+          # args: ["-feature-gates", "Upgrade=false,ResizePVC=false"]
+
+          # different new features are commented out below
+          # uncomment them to enable the featureGate
+          args:
+            - feature-gates
+            - Upgrade=false
+            # the below log level accepts "info" "debug" "warn" or "error"
+            - -zap-log-level
+            - info
+              # - debug
+          env:
+            - name: WATCH_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+            - name: OPERATOR_NAME
+              value: cockroachdb
+{{- range .CrdbVersions }}
```

Comment:
Can we add a comment of what all of these values are?

Reply:
Added.